### PR TITLE
Update the XCCDF HTML Guide XSLT

### PR DESCRIPTION
**Pull request by redhatrises**
- Remove right leaning content
- Bring rules in line with some of the look and feel of XCCDF HTML Report

### CODE REVIEW

**Comment by redhatrises** [comment]
This is not fully functioning but wanted to let people see it before continuing.

**Comment by mpreisler** [comment]
Love this, looking forward to the finished product.

**Comment by mpreisler** [comment]
The link to go to the start of the checklist doesn't work for me btw.

**Comment by redhatrises** [comment]
> The link to go to the start of the checklist doesn't work for me btw.
>
> And the group by combobox doesn't do anything on my end. I tried Chromium and Firefox.

@mpreisler working on that right now. Just wanted to start getting it out there visually for people to look at it and provide suggestions if they wanted or had any.

**Comment by mpreisler** [comment]
Ah, ok, cool. It looks good :-)

**Comment by redhatrises** [comment]
Need a little help with the JS functions in `openscap.js` for `guideSearch()` and `groupGuideBy(value)`... if anyone wants to help write those.

**Comment by mpreisler** [comment]
Based on discussion today I want to disambiguate some things here.

We have `ruleSearch` js function that searches through rules in HTML report. Since the HTML guide structure is different we need a similar function `guideSearch` that will search through the guide DOM.

Same thing for `groupRulesBy` for HTML report and `groupGuideBy` for HTML guide.

**Comment by mpreisler** [comment]
@openscap-ci test this please

**Comment by mpreisler** [comment]
@openscap-ci test this please

wtf is happening again with jenkins

File: xsl/xccdf-guide-impl.xsl [hunk 1]
```diff
@@ -170,77 +192,91 @@ Authors:
...
+                                            <xsl:with-param name="benchmark" select="$item/ancestor::cdf:Benchmark"/>
+                                            <xsl:with-param name="profile" select="$profile"/>
+                                        </xsl:apply-templates>
+                                    </div>
+                                </div>
+                            </xsl:for-each>
+                        </td></tr>
+
+                        <xsl:if test="$item/cdf:rationale">
+                            <tr><td><span class="label label-primary">Rationale:</span></td><td><div class="rationale">
```
Comment (mpreisler) [inline xsl/xccdf-guide-impl.xsl]: this TD element needs colspan=2

**Comment by mpreisler** [comment]
ACK

**Pull request merged**